README.md

```diff
@@ -1,12 +1,39 @@
 # registry
-Devfile registry - storing contents of the common devfile registry that feeds into the OCI based common registry
+Devfile registry - storing contents of the public devfile registry hosted at https://registry.devfile.io.
 
-## Build
+The public registry is updated weekly, by 12pm EST Wednesdays, with any updated stacks in this repository.
 
-To build this devfile registry into a container image:
+## Registry Updates
 
-1. git clone `github.com/devfile/registry-support`
-2. cd into `registry-support/build-tools`
-3. Run `build.sh <path-to-this-repository-on-disk>`
+The staging devfile registry, https://registry.stage.devfile.io is refreshed upon each commit to master in this repository. Production, https://registry.devfile.io, promoted manually and as mentioned above, is done each Wednesday, as needed.
 
-From there, push the container image to a container registry of your choice.
+If you are a stack owner and need to request an urgent refresh of https://registry.devfile.io before Wednesday (for example if a stack is broken), please open an issue in the [devfile/api](https://github.com/devfile/api) repository outlining the following:
```

Comment:
with the label `area/registry`

Reply:
Updated to say: "`/area registry` in the issue description" so that non-devfile org members are able to add the string without being a member.